src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class WriterServiceImpl implements WriterService {
+
+    public void writeToFile(String outputPath, String data) {
```

Comment:
It's recommended to use `@Override` annotation when implementing a method from an interface.